src/components/SubscriptionForm.js

```diff
@@ -1,29 +1,45 @@
-import React from 'react';
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux'
+import { setSubscribed } from '../actions'
 
-class SubscriptionForm extends React.Component {
-  constructor(){
-    super();
-    this.state = { value: '', submitted: false }
-    this.handleChange = this.handleChange.bind(this);
-    this.handleSubmit = this.handleSubmit.bind(this);
-  }
-
-  handleChange(e) {
-    this.setState({value: e.target.value});
-  }
-
-  handleSubmit(e) {
-    e.preventDefault();
-  }
-
-  render() {
+const SubscriptionForm = ({handleSubmit, subscribed}) => {
+  if(subscribed){
```

Comment:
is there a better way to break down this component to handle if logic in jsx or is this proper?